torch_frame/data/dataset.py

```diff
@@ -112,53 +140,44 @@ class DataFrameToTensorFrameConverter:
             column name into stats. Available as :obj:`dataset.col_stats`.
         target_col (str, optional): The column used as target.
             (default: :obj:`None`)
-        col_to_sep (Union[str, Dict[str, str]]): A dictionary or a string
-            specifying the separator/delimiter for the multi-categorical
-            columns. If a string is specified, then the same separator will
-            be used throughout all the multi-categorical columns. If a
-            dictionary is given, we use a separator specified for each
-            column. (default: :obj:`,`)
-        col_to_text_embedder_cfg (TextEmbedderConfig or dict, optional):
-            A text embedder configuration or a dictionary of configurations
-            specifying :obj:`text_embedder` that maps text columns into
-            :class:`torch.nn.Embeddings` and :obj:`batch_size` that
-            specifies the mini-batch size for :obj:`text_embedder`.
-            (default: :obj:`None`)
-        col_to_text_tokenizer_cfg (TextTokenizerConfig or dict, optional):
-            A text tokenizer configuration or dictionary of configurations
-            specifying :obj:`text_tokenizer` that maps sentences into a
-            list of dictionary of tensors. Each element in the list
-            corresponds to each sentence, keys are input arguments to
-            the model such as :obj:`input_ids`, and values are tensors
-            such as tokens. :obj:`batch_size` specifies the mini-batch
-            size for :obj:`text_tokenizer`. (default: :obj:`None`)
-        col_to_time_format (Union[str, Dict[str, str]], optional): A
-            dictionary or a string specifying the format for the timestamp
-            columns. See `strfttime documentation
+        col_to_sep (Dict[str, Optional[str]]): A dictionary specifying the
+            separator/delimiter for the multi-categorical columns.
+            (default: :obj:`{}`)
+        col_to_text_embedder_cfg (Dict[str, TextEmbedderConfig]):
+            A dictionary of configurations specifying :obj:`text_embedder` that
+            maps text columns into :class:`torch.nn.Embeddings` and
```

Comment:
I think it's not mapped to `:class:torch.nn.Embeddings`, let's change this.

Reply:
Good catch. Rewrote as follows:
:obj:`text_embedder` that embeds texts into vectors